.github/workflows/ROMROM-CHANGELOG-UPDATE.yaml

```
@@ -204,6 +234,7 @@ jobs:
                   # 동적 카테고리를 parsed_changes에 추가
                   for key, value in categories.items():
                       new_release["parsed_changes"][key] = value["items"]
+                      print(f"📝 parsed_changes에 추가: {key} -> {len(value['items'])}개 아이템")
                   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)